### PR TITLE
TickMeter: added FPS and AvgTime, improved docs, reformatted

### PR DESCRIPTION
**Pull request by mshabunin**
- New `TickMeter` methods: `getFPS`, `getAvgTimeSec` and `getAvgTimeMilli`
- Extracted code snippets from documentation
- Minor formatting fixes

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under OpenCV (BSD) License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [x] There is reference to original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: modules/core/include/opencv2/core/utility.hpp [hunk 1]
```diff
@@ -282,107 +282,100 @@ CV_EXPORTS_W double getTickFrequency();
...
+            return 0.;
+        return getTimeSec() / counter;
+    }
+
+    //! returns average time in milliseconds
+    CV_WRAP double getAvgTimeMilli() const
+    {
+        if (counter <= 0)
+            return 0.;
+        return getTimeMilli() / counter;
```
Comment (alalek) [inline modules/core/include/opencv2/core/utility.hpp:370]: Lets reduce logic:
```
return getAvgTimeSec() * 1e3;
```

Comment (mshabunin) [inline modules/core/include/opencv2/core/utility.hpp:370]:
```suggestion
        return getAvgTimeSec() * 1e3 / counter;
```

Comment (alalek) [reply modules/core/include/opencv2/core/utility.hpp]: `if` should be removed too.

Comment (mshabunin) [reply modules/core/include/opencv2/core/utility.hpp]: 🤦

**Pull request merged**